### ISSUE TITLE
Scroll wheel not working

### ISSUE THREAD

**Issue by onpumpanewsock** [krassowski/Anki-Zoom#10]
On  Anki 2.19 with latest zoom 2.1 the mouse scroll wheel no longer controls zoom.

**Comment by onpumpanewsock** [comment]
@krassowski any chance this is a quick fix? Seems like it was the 2.1.19 update that broke it

**Comment by krassowski** [comment]
Not really, got other priorities. I would advocate for adding a native zoom into the Anki btw.
